README.md

```diff
@@ -1,8 +1,6 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
-
-
+For security purposes .env files have not been uploaded to GitHub. To connect to a local test and development database it will be necessary to create a .env-test and a .env-development file in the root directory. Each file will need to contain the test 'PGDATABASE=databaseName' (without quotes), replacing 'databaseName with the relevant name of the database. 
 
```

Comment:
Your readMe just needs a slight tweak. You have your env files as` .env-test` etc however this should be `.env.test` also you should give some clue as to where to find the database name for example `(see /db/setup.sql for the database names).`